.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,36 @@
+name: Publish Docker image
```

Comment:
Looks good. Should we tie this with semrel or some other release pipeline?

Reply:
Yup I am very new to GH actions if you have some hint would be really appreciated

Reply:
See this recent blog on semrel with gh actions:
https://levelup.gitconnected.com/using-semantic-release-with-github-actions-c30d197829f1

We've been using semrel for docker hub releases on ambianic edge images:
https://github.com/ambianic/ambianic-edge/blob/master/.github/workflows/main.yml

Reply:
Ok will check and fix merge issues